### ISSUE TITLE
Bind mount with --volume doesn't support relative source path

### ISSUE THREAD

**Issue by pixdrift** [projectatomic/buildah#587]
**Description**
When using the `--volume` option to bind mount a volume with `buildah run`, a failure is returned if the provided source path is a relative path. The same command is successful when an absolute path is used for the source.

**Steps to reproduce the issue:**
1. Confirm source directory
```
# pwd
/root
```
```
# ls -lad rpm
drwxr-xr-x. 2 root root 4096 Apr 15 14:20 rpm
```
2. Attempt to specify volume source using relative reference (multiple methods)
```
# buildah run --volume ./rpm:/rpm working-container whoami
container_linux.go:348: starting container process caused "process_linux.go:402: container init caused \"rootfs_linux.go:58: mounting \\\"/tmp/buildah410569340/rpm\\\" to rootfs \\\"/var/lib/buildah/storage/overlay/5f90399a12a28650695220cb364fec24348a0ee38b350f0ad32603e16de3b5fa/merged\\\" at \\\"/rpm\\\" caused \\\"stat /tmp/buildah410569340/rpm: no such file or directory\\\"\""
```
```
# buildah run --volume rpm:/rpm working-container whoami
container_linux.go:348: starting container process caused "process_linux.go:402: container init caused \"rootfs_linux.go:58: mounting \\\"/tmp/buildah637239923/rpm\\\" to rootfs \\\"/var/lib/buildah/storage/overlay/5f90399a12a28650695220cb364fec24348a0ee38b350f0ad32603e16de3b5fa/merged\\\" at \\\"/rpm\\\" caused \\\"stat /tmp/buildah637239923/rpm: no such file or directory\\\"\""
```
3. Specify volume source using absolute path
```
# buildah run --volume /root/rpm:/rpm working-container whoami
root
```

**Describe the results you received:**
Error from container_linux.go when initialising the container

**Describe the results you expected:**
The volume to mount cleanly

**Output of `rpm -q buildah` or `apt list buildah`:**

```
buildah-0.16-201804131656.git83d7d10.el7.x86_64
```

**Output of `buildah version`:**

```
Version:       0.16
Go Version:    go1.8.3
Image Spec:    1.0.0
Runtime Spec:  1.0.0
Git Commit:    83d7d10
Built:         Sun Apr 15 14:16:02 2018
OS/Arch:       linux/amd64
```

**Output of `cat /etc/*release`:**

```
CentOS Linux release 7.4.1708 (Core)
```

**Output of `uname -a`:**

```
3.10.0-693.21.1.el7.x86_64 #1 SMP Wed Mar 7 19:03:37 UTC 2018 x86_64 x86_64 x86_64 GNU/Linux
```

**Comment by umohnani8** [comment]
Yeah, we don't support relative paths as the host path for the `--volume` flag, same thing with docker build.
```
➜  sudo docker build --volume ../test-help.txt:/run/test -t testing .
Sending build context to Docker daemon 1.479 MB
Step 1/9 : FROM fnndsc/ubuntu-python3:latest
 ---> 9b1c091b1dcc
Step 2/9 : MAINTAINER fnndsc "dev@babymri.org"
create ../test-help.txt: "../test-help.txt" includes invalid characters for a local volume name, only "[a-zA-Z0-9][a-zA-Z0-9_.-]" are allowed. If you intented to pass a host directory, use absolute path
```
@rhatdan is this something we want to support? If not, I will open a PR to return a better error message.

**Comment by rhatdan** [comment]
So we can close this as notabug.  Reopen if you disagree.

**Comment by pixdrift** [comment]
Even if the decision is not to support it, I think the error message should be improved as @umohnani8 has suggested above.

**Comment by rhatdan** [comment]
Ok @umohnani8 Can you improve the error message. Missed that part of the discussion.

**Comment by umohnani8** [comment]
Fixed #643
